src/Listener.h

```diff
@@ -33,14 +34,16 @@ class Listener {
    void Reset();
    std::string ThreadID();
    std::unique_ptr<Rifle> CreateFeedbackShooter();
+   void StorePayloadIfNecessary(std::vector<std::string>& dataFromQueue);
```

Comment:
Can't be const because I call another function from within ```StorePayloadIfNecessary``` that does alter this vector